ampbench_cli

```diff
@@ -205,10 +249,11 @@ const make_api_sd_validation = (api_validate_sd_return) => {
     };
 };
 
+
 program
-    .arguments('<url>')
-    .action(function(url) {
-        console.log('Analyzing url: %s', url);
+    .arguments('<url> <cssAddr>')
+    .action(function(url, cssAddr) {
+        cssValidator(cssAddr);
         analyze(url);
```

Comment:
yes, I though this was what we wanted.
I can check first analyze(url) and if OK go to cssValidator(cssAddr) and there give nagios the status code.